Formula/skaffold.rb

```diff
@@ -0,0 +1,26 @@
+class Skaffold < Formula
+  desc "Easy and Repeatable Kubernetes Development"
+  homepage "https://github.com/GoogleCloudPlatform/skaffold"
+  url "https://github.com/GoogleCloudPlatform/skaffold.git",
+    :tag => "v0.1.0",
```

Comment:
We prefer building from a source tarball, to enable Homebrew's caching mechanisms

Reply:
Version 0.1.0 is marked as “pre-release”. Homebrew's core policy is to only ship stable releases: https://docs.brew.sh/Acceptable-Formulae.html

Reply:
Our build relies on information in the git repository. We could change our build process add them manually here, at the expense of complicating the brew build.

Pre-release didn't really mean anything to us so I've removed it from the release.